The LTO build failed with:

```
libtool: link: c++ -std=c++11 -std=c++17 -O2 -flto -I/packages/glib2_x86-2.81.0-3/.self/develop/headers/x86 -I/packages/glib2_x86-2.81.0-3/.self/develop/headers/x86/glib-2.0 -I/packages/glib2_x86-2.81.0-3/.self/develop/lib/x86/glib-2.0/include -I/packages/libpcre2_x86-10.45-1/.self/develop/headers/x86 -DNCURSES_WIDECHAR -I/packages/ncurses6_x86-6.5-3/.self/develop/headers/x86 -O2 -flto -o sciteco-minimal -Wl,--export-dynamic  ./.libs/libsciteco-base.a -L/packages/glib2_x86-2.81.0-3/.self/develop/lib/x86 -L/packages/ncurses6_x86-6.5-3/.self/lib/x86 /sources/sciteco-2.5.1/contrib/scintilla/bin/scintilla.a /sources/sciteco-2.5.1/contrib/lexilla/bin/liblexilla.a -lgmodule-2.0 -lglib-2.0 -lintl -lpthread -lncursesw
/boot/system/develop/tools/x86/bin/../lib/gcc/i586-pc-haiku/13.3.0/../../../../i586-pc-haiku/bin/ld: /boot/system/develop/lib/x86/start_dyn.o: in function `_start':
start_dyn.c:(.text+0x3c): undefined reference to `main'
collect2: error: ld returned 1 exit status
```

--- a/app-editors/sciteco/sciteco_curses-2.5.1.recipe
+++ b/app-editors/sciteco/sciteco_curses-2.5.1.recipe
@@ -1,0 +1,77 @@
+SUMMARY="Scintilla-based Text Editor and Corrector (curses interface)"
+DESCRIPTION="SciTECO is an interactive text editor and text processing language. \
+It is inspired by Video TECO and DEC TECO-11, but incorporates many unique new ideas. \
+In SciTECO's paradigm the user inserts into an immediately executed command-line \
+which in turn performs the text processing operations. \
+All operations can be undone by \"rubbing out\" (deleting) from the command-line. \
+
+This package contains the curses (terminal) version of SciTECO."
+HOMEPAGE="https://sciteco.fmsbw.de/"
+COPYRIGHT="2012-2026 Robin Haberkorn"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://sciteco.fmsbw.de/downloads/v$portVersion/sciteco-$portVersion.tar.gz
+	https://sourceforge.net/projects/sciteco/files/v$portVersion/sciteco-$portVersion.tar.gz"
+CHECKSUM_SHA256="cc99c6855f844f0514f2ed4879bf6a02f11eb489a3b77d88ad7f0bcfe1379fbf"
+SOURCE_DIR="sciteco-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	sciteco_curses$secondaryArchSuffix = $portVersion
+	cmd:grosciteco.tes = $portVersion
+	cmd:sciteco = $portVersion
+	cmd:tedoc.tes = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgmodule_2.0$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgmodule_2.0$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:groff
+	cmd:m4
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	"
+
+BUILD()
+{
+	# Once we support an --enable-lto site-config-option, we should rather use that.
+	export CFLAGS="-O2 -flto"
+	export CXXFLAGS="-O2 -flto"
+	export LDFLAGS="-O2 -flto"
+	export AR=gcc-ar
+	export NM=gcc-nm
+	export RANLIB=gcc-ranlib
+
+	runConfigure --omit-dirs "binDir" \
+		./configure --bindir=$prefix/bin --with-interface=ncurses
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+}
+
+TEST()
+{
+	make check
+}

